ccdaservice/serveccda.js

```diff
@@ -8,6 +8,9 @@
  */
 
 "use strict";
+
+const enableDebug = true;
```

Comment:
Are you wanting debug turned on by default here?

Reply:
I'll have more changes coming however, moving the xml save allows debug on all the time. Which is how I have ran it for a couple years until @stephenwaite started turning it off which is okay with me. Still, having access to the last ccda xml and json generated to file has helped users debug in production. Especially the json.